cloudformation/infrastructure/fargate-cluster.yaml

```diff
@@ -244,6 +249,8 @@ Resources:
                 Value: !Ref FullEnvironmentName
               - Name: S3_BUCKET_NAME
                 Value: !Ref S3BucketName
+              - Name: EXPORT_S3_BUCKET_NAME
```

Comment:
Seeing this name makes me want to refactor `S3_BUCKET_NAME` to describe the usage, too, but that's probably a separate iteration.